joins/files.txt

```diff
@@ -0,0 +1,2 @@
+documents.json.bz2
+documents-1k.json.bz2
```

Comment:
What is the purpose of this file?

Reply:
It's a leftover from a copy-paste. I'm not sure if Rally uses it, I'll try to just delete it.